src/components/atoms/ac-button/ac-button.scss

```diff
@@ -32,9 +32,14 @@ ac-button {
     user-select: none;
     -webkit-appearance: none;
 
+    & > i {
```

Comment:
Replace it to `& > [slot=icon-start], & > [slot=icon-end]` to support both SVG (AcFaIcon used internally) and I tags.

Reply:
Solved